plugins/channelrx/heatmap/heatmapsink.cpp

```diff
@@ -26,7 +26,6 @@
 
 HeatMapSink::HeatMapSink(HeatMap *heatMap) :
         m_scopeSink(nullptr),
-        m_heatMap(heatMap),
```

Comment:
Remove from constructor parameters if not needed

Reply:
ok